src/tv2-common/migrations/index.ts

```diff
@@ -125,6 +129,61 @@ export function AddGraphicToGFXTable(
 	}
 }
 
+export function mapGFXTemplateToDesignTemplate(versionStr: string, studio: string, from: string, to: string) {
+	return literal<MigrationStepShowStyle>({
+		id: `${versionStr}.mapGFXTemplateToDesignTemplate.${from}.${studio}`,
+		version: versionStr,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const gfxTemplates = (context.getBaseConfig(from) as unknown) as TableConfigItemGFXTemplates[] | undefined
+
+			const designTemplates = (context.getBaseConfig(to) as unknown) as TableConfigItemGFXDesignTemplates[] | undefined
+
+			if (!gfxTemplates || !gfxTemplates.length) {
+				return false
+			}
+
+			if (designTemplates && designTemplates.length) {
+				return false
+			}
+
+			return gfxTemplates.some(g => g.IsDesign)
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			const gfxTemplates = (context.getBaseConfig(from) as unknown) as TableConfigItemGFXTemplates[]
+			const designTemplates = (context.getBaseConfig(to) as unknown) as TableConfigItemGFXDesignTemplates[]
+
+			gfxTemplates.filter(g => g.IsDesign).map(g => designTemplates.push(g))
+
+			context.setBaseConfig(to, (designTemplates as unknown) as ConfigItemValue)
+		}
+	})
+}
+
+export function removeDesignChangesFromGFXTemplate(versionStr: string, studio: string, configId: string) {
```

Comment:
I would prefer if this and `mapGFXTemplateToDesignTemplate` were merged into a single migration. It's just safer. When `mapGFXTemplateToDesignTemplate` failed, this one got executed anyway and deleted all the designs from GFX Template.

Reply:
What if the `validate` check was more robust on this migration and checks that there are also values in the GfxDesignTemplate before removing the column?

Since moving the data and removing the data is two different responsibilities, I like the fact that they have been split. 
But I agree, we don't want to end up deleting our data without moving them first.

Reply:
As of now i've actually moved them together, since i thought it made sense that one wouldn't be used without the other. I do see the idea that they have different responsibilities, however. Should i continue with them put together?